docs/user/production-considerations/task-manager-health-monitoring.asciidoc

```diff
@@ -0,0 +1,101 @@
+--------------------------------------------------
+// KIBANA
+
+Monitoring the `_health` endpoint of each {kib} instance in the cluster is the recommended method of ensuring confidence in mission critical services such as Alerting and Actions.
+
+[float]
+[[task-manager-configuring-health-monitoring]]
+=== Configuring the Monitored Health Statistics
+
+The health monitoring api monitors Task Manager's performance out of the box, but certain performance considerination are deployment specific and are configurable as a result.
+
+Health thresholds are configurable via the <<task-manager-health-settings,`xpack.task_manager.monitored_task_execution_thresholds`>> setting.
+This configuration specifies the threshold of failed task executions which, once exceeded, sets a health status of `warn` or `error` on the corresponding task type execution status.
+This setting can be set as either the default level, which is applied to all task types in the system, or at a custom task type specific level. 
+
+By default, this setting is configured to mark the health of every task type as `warning` when it exceeds 80% failed executions, and as `error` at 90%.
+This value can be set to any number between 0 to 100, and a threshold is hit when the value *exceeds* this number.
+This means that you can avoid setting the status to `error` by setting the threshold at 100, or hit `error` the moment any task fails by setting the threshold to 0 (as it will exceed 0 once a single failure occurs).
+
+Custom configurations allow you to set lower thresholds for task types you might consider critical, such as alerting tasks, which you might wish to detect sooner in an external monitoring service.
+
+[source,yml]
+----
+xpack.task_manager.monitored_task_execution_thresholds:
+  default: # <1>
+    error_threshold: 70
+    warn_threshold: 50
+  custom:
+    "alerting:.index-threshold": # <2>
+      error_threshold: 50
+      warn_threshold: 0
+----
+<1> default configuration, setting a system wide `warn` threshold at 50%, and `error` at 70% failure rate
+<2> custom configuration for the `alerting:.index-threshold` task type, setting a system wide `warn` threshold at 0% (which will set a `warn` status the moment any task of that type fails), and `error` at 50% failure rate
+
+[float]
+[[task-manager-consuming-health-stats]]
+=== Consuming Health Stats
+
+The `health` api is best consumed by via the `/api/task_manager/_health` endpoint.
+
+Additionally, the metrics are logged out into the {kib} `DEBUG` logger at a regular cadence.
+If you wish to enable Task Manager DEBUG logging in your {kib} instance, you will need to add the following to your `Kibana.yml`:
+```
+logging:
+  loggers:
+      - context: plugins.taskManager
+        appenders: [console]
+        level: debug
+```
+
+Please bear in mind that these stats are logged as often as your <<task-manager-settings,`xpack.task_manager.poll_interval`>> setting, which means it could add substantial noise to your logs.
```

Comment:
```suggestion
These stats are logged based the number of milliseconds set in your <<task-manager-settings,`xpack.task_manager.poll_interval`>> setting, which means it could add substantial noise to your logs.
```